test/integration/framework/controlplane_utils.go

```diff
@@ -98,3 +98,8 @@ func SharedEtcd() *storagebackend.Config {
 	cfg.Transport.ServerList = []string{GetEtcdURL()}
 	return cfg
 }
+
+// DefaultAPIServerFlags returns the default flags used to run kube-apiserver on tests
+func DefaultAPIServerFlags() []string {
```

Comment:
nit: DefaultTestServerFlags

as we call the thing test server, not test apiserver.

Reply:
maybe also worth a put a oneline comment behind each saying why we want that?